java/test/jmri/jmrit/roster/swing/RosterFrameTest.java

```diff
@@ -41,6 +41,31 @@ public void remoteCalls(String[] args){
         Assert.assertNotNull("exists", frame );
     }
 
+    @Test
+    public void testIdentifyEnabled() {
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
+        RosterFrame frame = new RosterFrame(){
+            // for now, don't show the status bar in the test, so
+            // we can sort out how to initialize the ProfileManager
```

Comment:
I'll get something in place this week to initialize the ProfileManager with an empty activate profile.